### PR TITLE
made some minor changes in home page

### PR DESCRIPTION
**Pull request by bipin7yadav**
added  add to cart page and wishlist page

### CODE REVIEW

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+                })
+            }}
+            />
         </div>
         <div className="cartWish">
             <Link to="/SignUp" className="btn"><h2>👤 </h2></Link>
-            <Link to="/Wishlist" className="btn Rel"><h2 >❤️</h2> <span className="badge">3</span></Link>
-            <Link to="/MyCart" className="btn Rel"><h2>🛒CART<span className="badge">2</span></h2></Link>
+            <Link to="/Wishlist" className="btn Rel"><h2 >❤️</h2> {wish.length>0?<span className="badge">{wish.length}</span>:<span></span>}</Link>
+            {/* {cart.length>0?<span className="badge">{cart.length}</span>:<span></span>} */}
```
Comment (gopalbbharadva) [inline src/components/Navigation/Navigation.jsx]: Remove comments if not needed.

File: src/contexts/ProductContext.js [hunk 2]
```diff
@@ -18,6 +20,25 @@ const ProductListProvider = ({children  }) => {
         })();
     },[]);
```
Comment (gopalbbharadva) [inline src/contexts/ProductContext.js:22]: USe effect is useless , I guess. You can remove it.

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -1,93 +1,77 @@
...
+            <div className="con1">
+                {category.map((a) => {
+                    const {img,categoryName,type,payload}=a;
+                return (<>
+                    <div class="img-con">
+                        <img src={img} alt="./" class="img" />
+                        {/* <label className="label"><Link to="/AllProducts" onClick={()=>{dispatchFilter({
+                            type:{type},
+                            payload:{payload}
+                        })}}><span className="link">{categoryName}</span></Link></label> */}
```
Comment (gopalbbharadva) [inline src/pages/Home/Home.jsx:27]: Remove un necessary code.

**Review by gopalbbharadva — APPROVED** [review]
Functionality working fine.

**Review by someshtiwarii — APPROVED** [review]
other than few code changes, looks good to merge!

**Pull request merged**